### DISCUSSION TITLE
Disable the "saved file" notification

### DISCUSSION THREAD

**Discussion by 1nuno** [kiyoon/jupynium.nvim#117] [Q&A]
Hello there, I started using this plugin and its really wonderful. I was just wondering if there is a way to disable the notification it gives me every time I save the file as it can get really annoying because i type ```:w<CR>``` very frequently, like every 5s or so. Its a habit, I don't even think about it, just muscle memory at this point. I would appreciate if anyone can help me fix this or at least point me in the right direction somewhere in the documentation. Thank you in advance.

EDIT:

I completely forgot to add the notification I was mentioning. Heres an example:

[Jupynium]: Downloaded ipynb file to /home/1nuno/Desktop/university/2nd_semester/2023_2024/APRP/
projeto/Untitled.ipynb

**Comment by kiyoon** [comment]
Hi, if you install [nvim-notify](https://github.com/rcarriga/nvim-notify) the notification gets less intrusive (because it doesn't ask for a keypress). That might be what you want?

**Reply by 1nuno** [reply]
Hi man, thanks for answering so quick and also thanks for saving us from having to edit code directly on a jupyter notebook :).

I really don't want to install the notify plugin as I'm fine with where the notifications are. What I wanted was a way to turn them off altogether.

**Reply by kiyoon** [reply]
I'm glad you're enjoying it :)
I guess the only way is to make customising notifications possible, which can introduce too many settings to configure so I need to think of a way to do it more adequately.

**Comment by kiyoon** [comment]
Until the changes can be addressed, I'd suggest removing the lines yourself: https://github.com/kiyoon/jupynium.nvim/blob/a0cfec051f47b72f4501970e5103120dd71891c2/src/jupynium/events_control.py#L922-L925

**Comment by kiyoon** [comment]
Hi, #120 addressed this issue by adding the notify ignore setting.
